src/Samples/Widget/Widget.scss

```diff
@@ -0,0 +1,32 @@
+@import "azure-devops-ui/Core/override.scss";
```

Comment:
Should be able to use platformCommon instead of override since I don't see anything from override being used here (for both scss files)

Reply:
Fixed.